singularity-eos/eos/eos_builder.hpp

```diff
@@ -33,6 +33,7 @@ namespace EOSBuilder {
 enum class EOSType {
   IdealGas,
   Gruneisen,
+  //  Vinet,
```

Comment:
Should we remove the commented out code?

Reply:
It's fine to leave or fine to remove. Slightly better to remove, but I consider this non-blocking.

Reply:
ok let's just remove the commented out code then